review/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+class Review(models.Model):
+    review = models.TextField()
+    dogowner_id = models.ForeignKey(DogOwner, on_delete=models.DO_NOTHING)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    rating = models.IntegerField(validators=[MinValueValidator(0), MaxValueValidator(5)])
+    creation_date = models.DateTimeField(auto_now_add=True)
+
+
+@receiver(post_save, sender=Review)
+def save_user_profile(sender, instance, **kwargs):
```

Comment:
Why this function is necessary?
You could just call save() where needed, isnt it?

Reply:
See comment bellow, we are not calling this function directly its just making sure that each save is validated